docs/watchlists-feeds-reports.rst

```diff
@@ -0,0 +1,244 @@
+Watchlists, Feeds, Reports, and IOCs
+====================================
+Watchlists are a powerful feature of Carbon Black Cloud Enterprise EDR. They allow an organization to set-and-forget
+searches on their endpoints' incoming events data, providing the administrator the opportunity to sift through high
+volumes of activity and focus attention on those that matter.
+
+**Note:** Use of these APIs requires that the organization be enabled for Enterprise EDR.  Verify this by logging into
+the Carbon Black Cloud Console, opening the menu in the upper right corner, and checking for an ``ENABLED`` flag
+against the "Enterprise EDR" entry.
+
+All examples here assume that a Carbon Black Cloud SDK connection has been set up, such as with the following code:
+
+::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> api = CBCloudAPI(profile='sample')
+
+Setting Up a Basic Custom Watchlist
```

Comment:
We jump to creating a Watchlist, it would be great to include a summary before that describes the objects so they can jump to the one that best fits their desired workflow.

Reply:
Adding this.